algolia/app.js

```diff
@@ -119,7 +119,7 @@ function parseDocsYml() {
                     code_snippets_link_map: new Object()
                 };
 
-                data.page_name = path.replace('docs/sources/', '');
+                data.page_name = path.replace('../sources/', '');
```

Comment:
I'm not sure how this was working before. This works for me now when I follow the instructions in the readme.